README.md

```diff
@@ -12,7 +12,7 @@ For any feedback, questions, or bugs, please file a Github issue, start a Github
 
 ## New in Version 3.x
 
-The official [cryoDRGN-ET](https://www.biorxiv.org/content/10.1101/2023.08.18.553799v1) release for heterogeneous subtomogram analysis. 
+The official [cryoDRGN-ET](https://www.biorxiv.org/content/10.1101/2023.08.18.553799v1) release for heterogeneous subtomogram analysis.
 
 * [NEW] Heterogeneous reconstruction of subtomograms. See documentation [on gitbook](https://ez-lab.gitbook.io/cryodrgn/)
```

Comment:
Add `cryodrgn filter`

Reply:
adding a separate section to describe the current minor release